rls/src/build/cargo.rs

```diff
@@ -282,30 +249,23 @@ fn run_cargo_ws(
         return Err(format_err!("error compiling dependent crate"));
     }
 
-    Ok(compilation_cx
+    return Ok(compilation_cx
```

Comment:
IIRC using a `return` as a last statement is considered un-idiomatic in Rust; I wouldn't change that here